irohad/ordering/impl/single_peer_ordering_service.cpp

```diff
@@ -112,9 +112,10 @@ namespace iroha {
             // In case of restart it reloads state.
             if (persistent_state_->createOsPersistentState() |
                 [this](const auto &state) {
-                  return state->saveProposalHeight(proposal_height_);
+                  return state->saveProposalHeight(proposal_height_+1);
```

Comment:
May it save twice `X` value and then make height equals `X + 2`?

Reply:
and put spaces around `+` plz

Reply:
That is interesting, do you have an idea how we can prevent this if the case with X in DB and X + 2 in memory is possible?